disputer/package.json

```diff
@@ -1,10 +1,25 @@
 {
-  "name": "disputer",
-  "version": "0.1.0",
+  "name": "@umaprotocol/disputer",
+  "version": "1.0.0",
   "description": "UMA Disputer",
   "private": true,
   "dependencies": {
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
+    "dotenv": "^6.2.0"
```

Comment:
Just want to learn more: shouldn't `dotenv` get installed by the root package.json?

Reply:
Resolved IRL: Adding any external imports to this list including `dotenv`, `winston` and `sinon`. All other imports are imported from other directories within this monorepo.